funcs/funcs.go

```diff
@@ -27,3 +34,57 @@ type PubSubHandler interface {
 	// HandleMessage processes a single PubSub message.
 	HandleMessage(ctx context.Context, msg *pubsub.Message) error
 }
+
+func RunHTTP(h WebhookHandler) {
+	ctx := context.Background()
+
+	defer report.Flush()
+	defer sentry.RecoverAndPanic(ctx)
+
+	if err := h.Init(); err != nil {
+		panic(err)
+	}
+	enabled, err := service.Register(ctx)
+	if err != nil {
+		panic(err)
+	}
+	if !enabled {
+		report.Info(ctx, "service disabled")
+		return
```

Comment:
I'm still concerned about what CloudRun will think if the server will just start and then stop immediately with exit status 0. We may need to change the `Run*` functions to return a bool value and test for it in the codegen:

```
func main() {
   if !cloudrun.RunHTTP(&f) {
    os.Exit(1)
   }
}
```

Reply:
Done.